streams/src/test/java/org/apache/kafka/streams/processor/internals/StateDirectoryTest.java

```diff
@@ -190,24 +220,34 @@ public void shouldCreateDirectoriesIfParentDoesntExist() throws Exception {
 
     @Test(expected = OverlappingFileLockException.class)
     public void shouldLockGlobalStateDirectory() throws Exception {
-        final FileChannel channel = FileChannel.open(new File(directory.globalStateDir(),
-                                                              StateDirectory.LOCK_FILE_NAME).toPath(),
-                                                     StandardOpenOption.CREATE, StandardOpenOption.WRITE);
         directory.lockGlobalState(1);
-        channel.lock();
+
+        try (
+            final FileChannel channel = FileChannel.open(
+                new File(directory.globalStateDir(), StateDirectory.LOCK_FILE_NAME).toPath(),
+                StandardOpenOption.CREATE,
+                StandardOpenOption.WRITE)
+        ) {
+            channel.lock();
+        } finally {
+            directory.unlockGlobalState();
+        }
     }
 
     @Test
     public void shouldUnlockGlobalStateDirectory() throws Exception {
-        final FileChannel channel = FileChannel.open(new File(directory.globalStateDir(),
-                                                              StateDirectory.LOCK_FILE_NAME).toPath(),
-                                                     StandardOpenOption.CREATE, StandardOpenOption.WRITE);
         directory.lockGlobalState(1);
-
         directory.unlockGlobalState();
 
-        // should lock without any exceptions
-        channel.lock();
+        try (
+            final FileChannel channel = FileChannel.open(
+                new File(directory.globalStateDir(), StateDirectory.LOCK_FILE_NAME).toPath(),
+                StandardOpenOption.CREATE,
+                StandardOpenOption.WRITE)
+        ) {
```

Comment:
No. You can have `try-finally` block without catch -- don't want to so anything on an exception beside releasing the lock.